### PR TITLE
Refactor test

### PR DESCRIPTION
**Pull request by alexander-akait**
**What kind of change does this PR introduce?**

refactor/tests

**Did you add tests for your changes?**

it is tests

**If relevant, did you update the documentation?**

No need

**Summary**

Part of https://github.com/webpack/webpack-cli/pull/2573 to avoid big changes

**Does this PR introduce a breaking change?**

No

**Other information**

In future we should migrate on stdout and stderr snapshot  tests everywhere, so feel free to send PRs with improve it, better do it by command, for `build` command by group of options, `serve` no need touch right now WIP on this

### CODE REVIEW

**Review by snitin315 — APPROVED** [review]

**Comment by alexander-akait** [comment]
We need search way to say `commander` use unlimited width for our tests

**Comment by anshumanv** [comment]
can you clarify?

**Comment by alexander-akait** [comment]
https://github.com/tj/commander.js/blob/master/index.js#L577 in our tests on some os we have `true` on other `false`, so snapshots are different due width of tty

File: packages/webpack-cli/lib/webpack-cli.js [hunk 1]
```diff
@@ -1023,7 +1023,7 @@ class WebpackCLI {
                     },
                     formatHelp: (command, helper) => {
                         const termWidth = helper.padWidth(command, helper);
-                        const helpWidth = helper.helpWidth || 80;
+                        const helpWidth = helper.helpWidth || process.env.WEBPACK_CLI_HELP_WIDTH || 80;
```
Comment (alexander-akait) [inline packages/webpack-cli/lib/webpack-cli.js:1026]: For testing purpose and for edge case where developer can set width of TTY

**Comment by alexander-akait** [comment]
Finally, I'll do a couple PRs for tests, improve them

File: packages/webpack-cli/lib/webpack-cli.js (hunk 1 above)
Comment (snitin315) [reply packages/webpack-cli/lib/webpack-cli.js:1026]: https://github.com/webpack/webpack.js.org/pull/4900
